app/src/main/java/cm/aptoide/pt/analytics/analytics/EventLogger.java

```diff
@@ -1,10 +1,29 @@
 package cm.aptoide.pt.analytics.analytics;
 
+import cm.aptoide.pt.ApplicationModule;
+import cm.aptoide.pt.FlavourApplicationModule;
 import java.util.Map;
 
 public interface EventLogger {
+  /**
+   * <p>Sends an {@code event} to the {@code EventLogger(s)} where that {@code event} is listed
+   * on.</p>
+   *
```

Comment:
Phrasing. Would go with something like "Sends an event to the corresponding EventLogger"